addons/rivet/api/rivet_packages.gd

```diff
@@ -0,0 +1,97 @@
+const _RivetResponse = preload("rivet_response.gd")
+
+## Lobby
+## @experimental
+class Lobby:
+	## Finds a lobby based on the given criteria. If a lobby is not found and
+	## prevent_auto_create_lobby is true, a new lobby will be created.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/find}[/url]
+	func find(body: Dictionary = {}):
+		return await Rivet.POST("matchmaker/lobbies/find", body).wait_completed()
+
+	## Joins a specific lobby. This request will use the direct player count
+	## configured for the lobby group.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/join}[/url]
+	func join(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/lobbies/join", body).wait_completed()
+
+	## Marks the current lobby as ready to accept connections. Players will not
+	## be able to connect to this lobby until the lobby is flagged as ready.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/ready}[/url]
+	func ready(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/lobbies/ready", body).wait_completed()
+
+	## If is_closed is true, the matchmaker will no longer route players to the 
+	## lobby. Players can still join using the /join endpoint (this can be disabled
+	## by the developer by rejecting all new connections after setting the lobby
+	## to closed). Does not shutdown the lobby.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/set-closed}[/url]
+	func setClosed(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.PUT("matchmaker/lobbies/set_closed", body).wait_completed()
+
+	## Creates a custom lobby.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/create}[/url]
+	func create(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.POST("matchmaker/lobbies/create", body).wait_completed()
+
+	## Lists all open lobbies.
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/list}[/url]
+	func list(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.GET("matchmaker/lobbies/list", body).wait_completed()
+
+	## 
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/set-state}[/url]
+	func setState(body: Dictionary = {}) -> _RivetResponse:
+		return await Rivet.PUT("matchmaker/lobbies/state", body).wait_completed()
+
+	## 
+	##
+	## [url]{https://rivet.gg/docs/matchmaker/api/lobbies/get-state}[/url]
+	func getState(body: Dictionary = {}) -> _RivetResponse:
```

Comment:
This has a path parameter in it. See https://rivet.gg/docs/matchmaker/api/lobbies/get-state